lib/utilities/reportHandler.js

```diff
@@ -98,19 +116,19 @@ function getCRRStats(log, cb) {
         const stats = {
             completions: {
                 count: completions.results.count,
-                size: parseFloat(completions.results.size) * 1000,
+                size: Math.round(parseFloat(completions.results.size) * 1000),
```

Comment:
Added `Math.round` because multiplying by 1000 for certain numbers still results in a floating point (e.g. `parseFloat('8.19') * 1000`).